docs/TutorialAsync.md

```diff
@@ -83,35 +121,57 @@ it('works with promises', () => {
 });
 ```
 
-We call `jest.mock('../request')` to tell Jest to use our manual mock. `it` expects the return value to be a Promise that is going to be resolved.
-You can chain as many Promises as you like and call `expect` at any time, as
-long as you return a Promise at the end.
-
-### `async`/`await`
-
-Writing tests using the `async`/`await` syntax is easy. Here is
-how you'd write the same example from before:
+You can chain as many Promises as you like and call `expect` at any time, as long as you return a Promise at the end. Or you can use the async/await syntax to achieve the same:
 
 ```js
-// async/await can also be used.
-it('works with async/await', async () => {
-  const userName = await user.getUserName(4);
-  expect(userName).toEqual('Mark');
+// 'async' causes the function to transaprently return a promise
+it('works with promises', async () => {
+  const name = await user.getUserName(5);
+  expect(name).toEqual('Paul');
 });
 ```
 
-To enable async/await in your project, install
-[`babel-plugin-transform-async-to-generator`](http://babeljs.io/docs/plugins/transform-async-to-generator/) or
-[`babel-preset-stage-3`](http://babeljs.io/docs/plugins/preset-stage-3/)
-and enable the feature in your `.babelrc` file.
+We must install and configure Jest before we can run the test. In the `package.json`, we need the `jest-cli` test runner, and the `babel-jest` adaptor. We also configure Jest to look at the code inside `src`, so that it doesn't get confused by the compiled code inside the `lib` folder:
+
+```
+npm install --save-dev jest-cli babel-jest
+```
+
+This is our final `package.json`, complete with the `test` script and `jest` config object:
+
+```json
+{
+  "dependencies": {
+    "babel-runtime": "*"
+  },
+  "devDependencies": {
+    "babel-cli": "*",
+    "babel-jest": "*",
+    "babel-plugin-transform-async-to-generator": "*",
+    "babel-plugin-transform-runtime": "*",
+    "babel-preset-es2015": "*",
+    "jest-cli": "*"
+  },
+  "scripts": {
+    "test": "jest",
+    "build": "babel src --out-dir lib"
+  },
+  "main": "lib/user",
+  "jest": {
+    "rootDir": "src"
```

Comment:
I'd prefer to say `testPathIgnorePatterns": ['build/.*']` rather than mess with rootDir.